src/CONST.ts

```diff
@@ -2715,6 +2756,23 @@ const CONST = {
                 SAGE_INTACCT_SYNC_IMPORT_SYNC_REIMBURSED_REPORTS: 'intacctImportSyncBillPayments',
             },
             SYNC_STAGE_TIMEOUT_MINUTES: 20,
+
+            // Map each connection to its designated display connection
+            get MULTI_CONNECTIONS_MAPPING() {
+                return {
+                    [this.NAME.QBO]: null,
+                    [this.NAME.QBD]: null,
+                    [this.NAME.XERO]: null,
+                    [this.NAME.NETSUITE]: this.NAME.NETSUITE,
+                    [this.NAME.NSQS]: this.NAME.NETSUITE,
+                    [this.NAME.SAGE_INTACCT]: null,
+                };
+            },
```

Comment:
This feels like over-engineering. Right now NetSuite and NSQS are the only connections that are linked in this way. I don't think we need a structure like this that is considering the rest of the connections don't have this paradigm.

Reply:
Agreed & Fixed